src/aihwkit/nn/conversion.py

```diff
@@ -113,3 +118,46 @@ def convert_to_analog(
         module._modules[name] = new_mod  # pylint: disable=protected-access
 
     return module
+
+
+def convert_to_analog_mapped(
+        module: Module,
+        rpu_config: RPUConfigGeneric,
+        realistic_read_write: bool = False,
+        weight_scaling_omega: float = 0.0,
+) -> Module:
+    """Convert a given digital model to analog counter parts with tile
```

Comment:
Change to: Convert a given digital model to its analog counterpart with tile mapping support.